src/components/MetricsPane/SelectedSiteMetrics.jsx

```diff
@@ -76,107 +89,111 @@ export const SelectedSiteMetrics = ({ selectedSampleEvent, view, setSelectedSamp
-      ) : (
-        <>
-          <SelectedSiteMetricsCardContainer>
-            <BiggerIconPersonCircle />
-            <SelectedSiteContentContainer>
-              <StyledHeader>Management Regime</StyledHeader>
-              <span>{selectedSampleEvent.management_name}</span>
-            </SelectedSiteContentContainer>
-          </SelectedSiteMetricsCardContainer>
-          <SelectedSiteMetricsCardContainer>
-            <BiggerIconUser />
-            <SelectedSiteContentContainer>
-              <StyledHeader>Admins</StyledHeader>
-              <span>{sampleEventAdmins}</span>
-            </SelectedSiteContentContainer>
-          </SelectedSiteMetricsCardContainer>
-          <SelectedSiteMetricsCardContainer>
-            <BiggerIconGlobe />
-            <SelectedSiteContentContainer>
-              <StyledHeader>Organizations</StyledHeader>
-              <span>{sampleEventOrganizations}</span>
-            </SelectedSiteContentContainer>
-          </SelectedSiteMetricsCardContainer>
-          <SelectedSiteMetricsCardContainer>
-            <StyledSvgContainer>
-              <img src={coralReefSvg} alt="coral reef" />
-            </StyledSvgContainer>
-            <SelectedSiteContentContainer>
-              <StyledHeader>Reef Habitat</StyledHeader>
-              <StyledReefContainer>
-                <StyledReefRow>
-                  <StyledReefItemBold>Reef Zone</StyledReefItemBold>
-                  <StyledReefItem>{selectedSampleEvent.reef_zone}</StyledReefItem>
-                </StyledReefRow>
-                <StyledReefRow>
-                  <StyledReefItemBold>Reef Type</StyledReefItemBold>
-                  <StyledReefItem>{selectedSampleEvent.reef_type}</StyledReefItem>
-                </StyledReefRow>
-                <StyledReefRow>
-                  <StyledReefItemBold>Reef Exposure</StyledReefItemBold>
-                  <StyledReefItem>{selectedSampleEvent.reef_exposure}</StyledReefItem>
-                </StyledReefRow>
-              </StyledReefContainer>
-            </SelectedSiteContentContainer>
-          </SelectedSiteMetricsCardContainer>
-          <SelectedSiteMetricsCardContainer>
-            <BiggerIconText />
-            <SelectedSiteContentContainer>
-              <StyledHeader>Notes</StyledHeader>
-              <span>{selectedSampleEvent.site_notes}</span>
-            </SelectedSiteContentContainer>
-          </SelectedSiteMetricsCardContainer>
-        </>
-      )}
-    </>
-  ) : null
+
+  const selectedSiteBody =
+    showMobileExpandedMetricsPane || isDesktopWidth ? (
+      <>
+        <SelectedSiteMetricsCardContainer>
+          <BiggerIconTextBoxMultiple />
+          <SelectedSiteContentContainer>
+            <StyledHeader>Project</StyledHeader>
+            <span>
+              {selectedSampleEvent.project_name}{' '}
+              {userIsMemberOfProject(selectedSampleEvent.project_id, mermaidUserData) ? (
+                <IconPersonCircle />
+              ) : null}
```

Comment:
Would `&&` work here instead of a ternary?

Reply:
I didnt write this code, just converted it from a nested component to jsx, BUT, I think ternaries should be the default for conditional rendering due to logical quirks with `&&` which can create bugs. 

https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx